CactusTutorial.ipynb

```diff
@@ -535,11 +545,117 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "# start simulation, watch log output\n",
-    "!./simfactory/bin/sim create-run tov_ET \\\n",
+    "%%bash\n",
+    "# start simulation\n",
+    "./simfactory/bin/sim create-submit tov_ET \\\n",
     "  --parfile=par/tov_ET.par --procs=2 --num-threads=1 --walltime=0:20:0"
```

Comment:
This does not work for me without specifying --ppn-used=2.

Reply:
same as above.